app/views/issues/index.html.erb

```diff
@@ -60,6 +61,13 @@
           <td class="text-nowrap"><%= link_to t(".reports_count", :count => issue.reports_count), issue %></td>
           <td><%= link_to reportable_title(issue.reportable), reportable_url(issue.reportable) %></td>
           <td><%= link_to issue.reported_user.display_name, issue.reported_user if issue.reported_user %></td>
+          <td>
+            <% unique_reporters = issue.reports.uniq { |report| report.user.id } %>
+            <% unique_reporters[..2].each_with_index do |report, index| %>
```

Comment:
This could be done more simply with something like:

```
safe_join(unique_reporters[..2].map { |u| link_to u.display_name, u }, ",")
```

That's assuming the previous edit to make the reporter list a list of users.

Reply:
`safe_join` was used to join names list.